static/css/diff.less

```diff
@@ -115,4 +115,34 @@ ins {
 del {
   color: #e44028;
   text-decoration: line-through;
+}
+#back2Top {
+    width: 40px;
+    line-height: 40px;
+    overflow: hidden;
+    z-index: 999;
+    display: none;
+    cursor: pointer;
+    -moz-transform: rotate(270deg);
+    -webkit-transform: rotate(270deg);
+    -o-transform: rotate(270deg);
+    -ms-transform: rotate(270deg);
+    transform: rotate(270deg);
```

Comment:
I don't think most of these transforms are needed, in fact I think only webkit-transform is needed.

Browser support is pretty good these days.
https://caniuse.com/#search=transform